homeassistant/components/twinkly/light.py

```diff
@@ -112,11 +115,6 @@ def unique_id(self) -> str | None:
         """Id of the device."""
         return self._id
 
-    @property
-    def name(self) -> str:
-        """Name of the device."""
-        return self._name if self._name else "Twinkly light"
-
     @property
     def model(self) -> str:
         """Name of the device."""
```

Comment:
Unrelated to your PR, but just noticed it ^^

```suggestion
        """Model of the device."""
```